static/src/javascripts/es6/projects/common/modules/crosswords/comments.js

```diff
@@ -0,0 +1,15 @@
+import config from 'common/utils/config';
+import DiscussionLoader from 'common/modules/discussion/loader';
+import qwery from 'qwery';
+
+function loadComments() {
+
+    if (config.switches.discussion && config.page.commentable) {
+        const el = qwery('.discussion')[0];
+        if (el) {
+            new DiscussionLoader().attachTo(el);
+        }
+    }
+}
+
+export default { 'loadComments' : loadComments };
```

Comment:
this could be simplified as `export default loadComments`, or even move `export default` to before the function declaration